src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentItemExtensions.cs

```diff
@@ -6,6 +6,39 @@ namespace OrchardCore.ContentManagement
 {
     public static class ContentItemExtensions
     {
+        public static bool TryGet<TPart>(this ContentItem contentItem, out TPart part) where TPart : ContentPart
```

Comment:
I remember there's `As<Part>()` am I right?

Reply:
Yes there is As<>, we also have Get<> and GetOrCreate<>

Reply:
> Yes there is As<>, we also have Get<>

What a difference between those?

Reply:
GetOrCreate: Gets a content part by its type or create a new one.
Both Get<> and As<>: Gets a content element by its name. They are the same thing